monkey/monkey_island.py

```diff
@@ -10,16 +10,27 @@
 from monkey_island import config_loader  # noqa: E402
 from monkey_island.cc.server_utils.island_logger import setup_logging  # noqa: E402
 
+
+def create_data_dir(data_dir):
```

Comment:
Can we consolidate data dir creation with `monkey/monkey_island/cc/setup.py`? That's where the current workflow related to monkey island setup is. If not, can we move this function there at least?